main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: baby
+//  action 1: the baby cries
+baby.cry();
+//  action 2: the baby grabs
+baby.grab();
+//  action 3: the baby eats
+baby.eat();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bird
+//  action 1: the bird flies
+bird.fly();
+//  action 2: the bird sleeps
+bird.sleep();
+//  action 3: the bird pecks
+bird.peck();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bitwig
+//  action 1:  bitwig checks the contents of custom VST folder
+bitwig.customVstFolderContent();

```

Comment:
the pseudo-code function name has no verb in it.
`checks` is the verb in your plain-english description
- [ ] revise